openproblems/tasks/_batch_integration/batch_integration_embed/methods/baseline.py

```diff
@@ -0,0 +1,34 @@
+from .....tools.decorators import method
+from .....tools.utils import check_version
+
+import numpy as np
+
+
+@method(
+    method_name="No Integration",
+    paper_name="No Integration (baseline)",
+    paper_url="https://openproblems.bio",
+    paper_year=2022,
+    code_url="https://github.com/openproblems-bio/openproblems",
+    is_baseline=True,
+)
+def no_integration(adata, test=False):
+    adata.obsm["X_emb"] = adata.obsm["X_uni_pca"]
+    adata.uns["method_code_version"] = check_version("openproblems")
+    return adata
+
+
+@method(
+    method_name="Random Integration",
+    paper_name="Random Integration (baseline)",
+    paper_url="https://openproblems.bio",
+    paper_year=2022,
+    code_url="https://github.com/openproblems-bio/openproblems",
+    is_baseline=True,
+)
+def random_integration(adata, test=False):
+    adata.obsm["X_emb"] = adata.obsm["X_uni_pca"][
+        np.random.permutation(np.arange(adata.shape[0]))
+    ]
+    adata.uns["method_code_version"] = check_version("openproblems")
```

Comment:
Added now :)

Reply:
These `no_integration` and `random_integration` baseline integration methods do not make much sense to me, I guess the baseline model should be the most basic method that has at least minimal integration effects

Reply:
>  I think you added a random integration per batch, but not per cell type

Oops! You're absolutely right. Though I think I might keep both -- it might be also a useful baseline to scramble biology within batches but keep batches disconnected from one another.